[Azure AD roles] Restricted management administrative unit properties (Beta) (#16295)
microsoftgraph/microsoft-graph-docs: concepts/permissions-reference.md

| _Directory.Read.All_ |Read directory data | Allows the app to read data in your organization's directory, such as users, groups and apps. **Note**: Users may consent to applications that require this permission if the application is registered in their own organization’s tenant.| Yes | No |
| _Directory.ReadWrite.All_ |Read and write directory data | Allows the app to read and write data in your organization's directory, such as users, and groups. It does not allow the app to delete users or groups, or reset user passwords. | Yes | No |
| _Directory.AccessAsUser.All_ |Access directory as the signed-in user  | Allows the app to have the same access to information in the directory as the signed-in user. | Yes | No |

> | _Directory.Write.Restricted_ |Manage all restricted resources  | Allows an app to manage restricted resources as per existing permissions of the app, on behalf of the signed-in user. | Yes | No |

Reviewer: Align these with the official description that's surfaced to devs and admins.
<img width="373" alt="image" src="https://user-images.githubusercontent.com/14026935/172428140-f572f915-9a69-4523-a6a6-002561c4a302.png">

```suggestion
| _Directory.Write.Restricted_ |Manage restricted resources in the directory  | Allows the app to manage restricted resources based on the other permissions granted to the app, on behalf of the signed-in user. | Yes | No |
```

|:-----------------------------|:-----------------------------------------|:-----------------|:-----------------|
| _Directory.Read.All_ | Read directory data | Allows the app to read data in your organization's directory, such as users, groups and apps, without a signed-in user. | Yes |
| _Directory.ReadWrite.All_ | Read and write directory data | Allows the app to read and write data in your organization's directory, such as users, and groups, without a signed-in user. Does not allow user or group deletion. | Yes |

> | _Directory.Write.Restricted_ |Manage all restricted resources  | Allows an app to manage restricted resources as per existing permissions of the app, without a signed-in user. | Yes |

Reviewer: ```suggestion
| _Directory.Write.Restricted_ |Manage restricted resources in the directory  | Allows the app to manage restricted resources based on the other permissions granted to the app, without a signed-in user. | Yes |
```
